src/pages/SignUp/SignUp.js

```diff
@@ -23,15 +23,25 @@ export default class Login extends Component {
   };
 
   checkAll = e => {
-    const { name, email, pw, address } = this.state;
+    const { name, email, password, address } = this.state;
 
     if (
       chkName(name) &&
       chkEmail(email) &&
-      chkPwd(pw) &&
+      chkPwd(password) &&
       chkAddress(address) === true
     ) {
-      alert('회원가입 완료');
+      fetch('http://10.58.3.41:8000/accounts/signup', {
+        method: 'POST',
+        body: JSON.stringify({
+          name: name,
+          email: email,
+          password: password,
+          address: address,
+        }),
+      })
+        .then(alert('가입이 완료되었습니다.'))
+        .then(this.props.history.push('/login'));
```

Comment:
```suggestion
      fetch(`${API}/accounts/signup`, {
        method: 'POST',
        body: JSON.stringify({ name, email, password, address }),
      })
        .then((res) => {
          if (res.ok) {
            alert("가입이 완료되었습니다.")
            this.props.history.push('/login')
          }
        })
```
- 요청보내는 base url은 일반적으로 src/config.js에 분리해두어 코드의 관리포인틀을 줄여주는 것이 좋습니다.
- 객체의 키와 값의 변수명이 같을 경우 위와 같이 단축해줄 수 있습니다
- 현재 then을 두번 사용하고 계신데 then은 Promise로 다루는 비동기 값들을 동기적으로 다루기 위해 사용하는 것이라 (ex. res.json() 같은 동작이 다 끝난 후에 다른 로직을 실행시켜야 한다면 다음 로직은 .then 이후에 실행시켜야 함) 지금 같은 상황에서는 첫번째 then에서 서버로부터 받은 값이 유효하다는 체크만 해주고 성공 케이스 로직을 태우면 될 것 같습니다. 첫번째 then에서 넘어오는 res 값 직접 콘솔 찍어보세요!